pkg/webhook/resources/setting/validator.go

```diff
@@ -978,6 +980,29 @@ func (v *settingValidator) validateUpdateVolumeSnapshotClass(_ *v1beta1.Setting,
 	return v.validateVolumeSnapshotClass(newSetting)
 }
 
+func (v *settingValidator) validateUpdateLonghornV2DataEngine(oldSetting, newSetting *v1beta1.Setting) error {
+	if oldSetting.Value == newSetting.Value {
+		return nil
+	}
+
+	// check if any disk is still using for v2 data engine
+	if oldSetting.Value == "true" && newSetting.Value == "false" {
+		lhnodes, err := v.lhNodeCache.List(util.LonghornSystemNamespaceName, labels.Everything())
+		if err != nil {
+			return werror.NewInternalError(err.Error())
+		}
+		for _, node := range lhnodes {
```

Comment:
if there are multiple block device type disks, do we want to aggergate all names and return them together?